packages/jaeger-ui/src/utils/tracking/ga.test.js

```diff
@@ -96,33 +97,33 @@ describe('google analytics tracking', () => {
   describe('trackPageView', () => {
     it('tracks a page view', () => {
       tracking.trackPageView('a', 'b');
-      expect(calls).toEqual([['send', { hitType: 'pageview', page: 'ab' }]]);
+      expect(window.dataLayer).toEqual([['event', 'page_view', { page_path: 'ab' }]]);
     });
 
     it('ignores search when it is falsy', () => {
       tracking.trackPageView('a');
-      expect(calls).toEqual([['send', { hitType: 'pageview', page: 'a' }]]);
+      expect(window.dataLayer).toEqual([['event', 'page_view', { page_path: 'a' }]]);
     });
   });
 
   describe('trackError', () => {
     it('tracks an error', () => {
       tracking.trackError('a');
-      expect(calls).toEqual([
-        ['send', { hitType: 'exception', exDescription: expect.any(String), exFatal: false }],
+      expect(window.dataLayer).toEqual([
+        ['event', 'exception', { description: expect.any(String), fatal: false }],
```

Comment:
If there is no external schema and we control the names, changing them means we're introducing breaking changes for people who may already be using the existing integration, so their dashboards could break.

Reply:
There is no definite schema that these calls adhere to. You can find the [API specification here](https://developers.google.com/tag-platform/gtagjs/reference). There are also just some `recommended events` and `recommended parameters` for these events. 

I tried to look at the source of the `ReactGA` and the API specification shared above to figure out the possible actual `tag` calls the library made for us.

Reply:
I an judging by the fact that the unit test is changing, so it seems like you're changing the schema. Can we keep it the same as before?

Reply:
Earlier, the unit tests tested what calls were being made to the `ReactGA` library. What calls, in turn, the same made to the `gtag` library was completely abstracted away from us.
Now the tests are checking what `gtag` calls are being made. Since we are losing the `ReactGA` library altogether, the tests can't be kept the same.